### PR TITLE
pbwt support

### PR DESCRIPTION
**Pull request by tiagoantao**
This supports pbwt
----------------

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pbwt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  skip: true # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - htslib
+    - zlib
+  run:
+   - htslib
```
Comment (bgruening) [inline recipes/pbwt/meta.yaml]: I think those can be removed now, they export themself with run-exports nowadays.

File: recipes/pbwt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+   - zlib
+
+test:
+  commands:
+    - pbwt > /dev/null
```
Comment (bgruening) [inline recipes/pbwt/meta.yaml]: /dev/null should not be needed

File: recipes/pbwt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+
+about:
+  home: https://github.com/richarddurbin/pbwt
+  license: GPL
```
Comment (bgruening) [inline recipes/pbwt/meta.yaml]: can you please add license_file and license_family ... if it is GPL we need to ship a license_file ... strictly ... or just ignore me ;)

Comment (tiagoantao) [reply recipes/pbwt/meta.yaml]: I actually had the license wrong :(

**Comment by bgruening** [comment]
Merci.

**Pull request merged**